src/test/java/bench/BenchmarkRunner.java

```diff
@@ -0,0 +1,19 @@
+package bench;
+
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+public class BenchmarkRunner {
+
+    public static void main(String[] args) throws RunnerException {
```

Comment:
You should be able to run the benchmarks from IDEa by running this method. Good thing is that you should also be able to debug.